components/scream/src/share/util/scream_time_stamp.cpp

```diff
@@ -116,13 +116,21 @@ double TimeStamp::get_julian_day () const {
 
 double TimeStamp::get_julian_day (const int yy, const int mm, const int dd, const double ss) const {
   // Initialize Julian Day
-  double julianday = (dd+1) + ss;
+  double julianday = dd + ss;
```

Comment:
I just noticed: this adds things quite unrelated: the day of the month and the seconds of the day. Perhaps you wanted to do something like `double julianday = dd*3600 + ss;`?

Reply:
ahh yes, good catch.  I think I was thrown off by ss being a double that maybe it was fraction of day.  But it's double to allow for `dt<1s`.  I've fixed this in the newest version to `dd+ss/86400` since we are calculating Julian "day" in units of days.

Reply:
@AaronDonahue This still seems wrong to me...